SQLStore: Prevent concurrent migrations (#44101)
grafana/grafana: docs/sources/administration/configuration.md

Sets the maximum amount of time a connection may be reused. The default is 14400 (which means 14400 seconds or 4 hours). For MySQL, this setting should be shorter than the [`wait_timeout`](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_wait_timeout) variable.

### locking_attempt_timeout_sec

> For "mysql" only if `lockingMigration` feature toggle is set. How many seconds to wait before failing to lock the database for the migrations, default is 0.

Reviewer: ```suggestion
For "mysql", if `lockingMigration` feature toggle is set, specify the time (in seconds) to wait before failing to lock the database for the migrations. Default is 0.
```
```suggestion
For "mysql" only if `lockingMigration` feature toggle is set. How many seconds to wait before failing to lock the database for the migrations, default is 0.
```
